tests/integration/_experimental/search/test_embeddings.py

```diff
@@ -44,8 +50,8 @@
         np.array([], dtype=np.float16),
     ],
 )
-def test_upload_embeddings(embedding: np.ndarray) -> None:
-    test_case_name = with_test_prefix(f"{__file__} test_upload_embeddings")
+def test__upload_embeddings(embedding: np.ndarray) -> None:
+    test_case_name = with_test_prefix(f"{__file__} test_upload_embeddings {uuid.uuid4()}")
```

Comment:
For my own knowledge, why is this additional uuid needed?

Reply:
I don't remember the specific error code, but iirc this test was intended to repeat on the 4 different arguments of embedding, and the first one would be successful while the rest would fail, complaining that a test case with this name already exists. An alternative approach is to re-use the test case as a fixture, similar to what I did with `dataset_name` below.